packages/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy/lib/LinkQueueLogger.ts

```diff
@@ -0,0 +1,209 @@
+import type { ILinkQueue, ILink } from '@comunica/bus-rdf-resolve-hypermedia-links-queue';
+import { LinkQueueWrapper } from '@comunica/bus-rdf-resolve-hypermedia-links-queue';
+import type { Logger } from '@comunica/types';
+import { PRODUCED_BY_ACTOR } from '@comunica/types-link-traversal';
+import type { Algebra } from 'sparqlalgebrajs';
+
+/**
+ * A link queue that log information about event happening in the queue
+ */
+export class LinkQueueLogger extends LinkQueueWrapper {
+  public readonly query: Algebra.Operation;
```

Comment:
Is query really needed?

Reply:
If we use the same engine (or the same process) is used for multiple queries it will be hard to determine to which query the link is information is for.